components/logins/src/store.rs

```diff
@@ -49,6 +57,23 @@ fn create_sync_engine(
     }
 }
 
+fn map_bulk_result_entry(
+    enc_login: Result<EncryptedLogin>,
+    encdec: &dyn EncryptorDecryptor,
+) -> BulkResultEntry {
+    match enc_login {
+        Ok(enc_login) => match enc_login.decrypt(encdec) {
+            Ok(login) => BulkResultEntry::Success { login },
+            Err(error) => BulkResultEntry::Error {
+                message: error.to_string(),
+            },
+        },
+        Err(error) => BulkResultEntry::Error {
+            message: error.to_string(),
+        },
```

Comment:
This branch indicates that we can't decrypt the data we just encrypted right?  I think you should log a warning -- it seems like it's an indication of a bigger issue.

Reply:
Changed the logging to a `warn!` after rebasing